lib/components/CentralDevice.jsx

```diff
@@ -133,6 +136,32 @@ class CentralDevice extends React.PureComponent {
         const advIconTitle = advertising ? 'Advertising' : 'Not advertising';
         const iconCheckmarkConnUpdate = autoConnUpdate ? 'icon-ok' : '';
         const iconCheckmarkPairing = (security && security.autoAcceptPairing) ? 'icon-ok' : '';
+        const deviceFamily = familyDefinitions[deviceInfo.family].split('_')[0].replace('NRF', 'nRF');
+        const deviceType = deviceTypeDefinitions[deviceInfo.deviceType].split('_')[0].replace('NRF', 'nRF');
+        const softdeviceVersion = versionInfo.sdBleApiVersion;
+
+        const localDeviceInfoItems = (() => {
+            const items = [];
+            items.push(<MenuItem key="deviceFamily" header>
+                Family:
+            </MenuItem>);
+            items.push(<MenuItem key="deviceFamilyValue">
+                {deviceFamily}
+            </MenuItem>);
+            items.push(<MenuItem key="deviceType" header>
+                Type:
+            </MenuItem>);
+            items.push(<MenuItem key="deviceTypeValue">
+                {deviceType}
+            </MenuItem>);
+            items.push(<MenuItem key="softdeviceVersion" header>
+                SoftDevice Version:
+            </MenuItem>);
+            items.push(<MenuItem key="softdeviceVersionValue">
+                {softdeviceVersion}
+            </MenuItem>);
+            return items;
+        })();
```

Comment:
Having this as menu items does not seem like the right approach. The values can be selected in the menu, but nothing happens when selecting a value.

Currently, the opened adapter is shown with "nRF5x":

![image](https://user-images.githubusercontent.com/461755/32767412-993cae74-c913-11e7-9ea0-a9ee76d8d2f8.png)

Could we perhaps put the device family (e.g. nRF52) or device type (e.g. nRF52832) here, and remove the menu items?

Reply:
I am not sure if there will be more info that are going to be listed here. Need a discussion.
In my opinion, the "nRF5x" can be replaced by the family. 
And I will try to avoid using menu here if the information list is still needed.

Reply:
Agreed, we need to think twice before adding new UI elements to avoid cluttering the interface. If we can convey the information needed without adding more elements, that would be best. 
I think showing the device type (nRF52832) should be sufficient, and log the rest with logger.info() so it is visible in the log view.

Reply:
In hindsight, before adding new UI elements, it is wise to have the UI discussion *before* starting implementing, preferably with sketches or drawings.

Reply:
Yes, agreed.